internal/ui/baricon.go

```diff
@@ -138,7 +138,7 @@ func (bi *barIcon) TappedSecondary(*fyne.PointEvent) {
 
 	c := fyne.CurrentApp().Driver().CanvasForObject(bi)
 	pos := fyne.CurrentApp().Driver().AbsolutePositionForObject(bi)
-	menu = widget.NewPopUpMenuAtPosition(fyne.NewMenu("", []*fyne.MenuItem{addRemove}...), c, pos)
+	widget.ShowPopUpMenuAtPosition(fyne.NewMenu("", []*fyne.MenuItem{addRemove}...), c, pos)
```

Comment:
Sorry, I think this breaks the `menu.Hide` code in the `addRemove` callback. There is no reference to the menu any more

Reply:
Good catch. It is a kind of strange way to do the code, but I have reverted this for now.